### ISSUE TITLE
Accessibility: The questions and their response buttons should be grouped with <fieldset> and <legend>

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3082]
![image](https://user-images.githubusercontent.com/1005609/78628091-13008100-7848-11ea-9549-42c2d5c0a002.png)

**Actual**
A screen reader user with tab navigation just hears the response values such as “Yes” or “No” without hearing the question.  A screen reader user has to use arrow keys to read the corresponding question which is not a desired experience.
 
**Expected**
Use <fieldset> and <legend> to associate/bind questions and their corresponding response buttons. 
 
**WCAG Violation**
Info and Relationships 1.3.1
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Parsing 4.1.1
https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by compulim** [comment]
This is fixed in #3076.
